x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -244,19 +244,26 @@ export const getManagementFilteredLinks = async (
   plugins: StartPlugins
 ): Promise<LinkItem> => {
   const fleetAuthz = plugins.fleet?.authz;
-  const { endpointRbacEnabled, endpointRbacV1Enabled } = ExperimentalFeaturesService.get();
-  const endpointPermissions = calculatePermissionsFromCapabilities(core.application.capabilities);
+  const { endpointRbacV1Enabled } = ExperimentalFeaturesService.get();
```

Comment:
Why the `endpointRbacEnabled` has been removed?

Reply:
Since this HIE logic is for `rbacV1` now based on the Host Isolation privilege, I thought it best to remove the flag to avoid ambiguity.